src/components/Catalog/GraphItem/GraphItem.tsx

```diff
@@ -37,7 +38,7 @@ const GraphItem = ({ graphId, title, type }: GraphItemProps) => {
             onMouseLeave={() => setIsHovered(false)}
             onDoubleClick={() => addWidget(graphId)}
         >
-            <Icon className="graph-icon" />
+            <GraphTypeIcon className="graph-icon" />
             <div className='title'>{title}</div>
             {isHovered && <DeleteIcon className='delete-icon' onClick={deleteGraph} />}
         </div>
```

Comment:
This comment is not related to this file, I just can't comment on the other one -
there is an empty file called GraphContainer.scss